soccer/gameplay/plays/Attack_Goal/adaptive_attack.py

```diff
@@ -0,0 +1,292 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+
+import standard_play
+import evaluation.ball
+import evaluation.passing_positioning
+import tactics.coordinated_pass
+import tactics.defensive_forward
+import tactics.simple_zone_midfielder
+import tactics.advance_zone_midfielder
+import skills.move
+import skills.capture
+import situational_play_selection
+
+
+class AdaptiveAttack(standard_play.StandardPlay):
```

Comment:
Adaptive attack is adaptive formation with all clearing functionality removed. I should probably add that as a comment. I've created new files for the variants to keep them separate from the known working and all situation covering adaptive formation, which has been moved to the Legacy Folder.

Reply:
Bump for adding the comment to the code